index.html

```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
+            </div>
+            <div class="card_details">
+              <ul>
+                  <li class="canopy">CANOPY</li>
+                   <li>Back End Dev</li>
+                   <li>2015</li>
+              </ul>
+               <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+           </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
+          <div class="works-card">  
+            <div class="">
+              <img src="static/images/Snapshoot Portfolio 3.png" alt="">
+            </div>
+            <div>
+              <h2 class="works-title">Tonic</h2>
+            </div>
+            <div class="card_details">
+              <ul>
+                  <li class="canopy">CANOPY</li>
+                   <li>Back End Dev</li>
+                   <li>2015</li>
+              </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
+          <div class="works-card">
+            <div>
+              <img src="static/images/Snapshoot Portfolio 4.png" alt="">
+            </div>
+            <div>
+              <h2 class="works-title">Multi-Post Stories</h2>
+              <div class="card_details">
+                <ul>
+                    <li class="canopy">CANOPY</li>
+                     <li>Back End Dev</li>
+                     <li>2015</li>
+                </ul>
+               
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
+          <div class="works-card-2">
+              <h2 class="works-title">About Myself</h2>
+              <P class="about-me">Hello I'm a software developer I can help you build a project, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded don't hesitate to contact me.</P>
+              <p class="icon">LET'S CONNECT</p>
```

Comment:
- [ ] This must be a `h` header tag as this is for best semantic practices.
- If there are other titles (e.g., "Let's connect") use a header tag `hx`